lib/index.js

```diff
@@ -239,6 +271,11 @@ export default class HyperList {
     const estTo = from + this[_cachedItemsLen];
     const to = estTo > total ? total : estTo < 0 ? 0 : estTo;
 
+    const fastCache = [];
```

Comment:
`from` is wrong here because `this[_screenItemsLen]` doesn't account the dynamic height.